.gitignore

```diff
@@ -26,3 +26,5 @@
 .envrc
 /inventory.yaml
 /spec/fixtures/litmus_inventory.yaml
+/acceptance/junit
+/acceptance/log
```

Comment:
I don't know if this file is pdk synced or not (if it is, it should probably not be edited), but please add the missing \n.

Reply:
Since I am not sure if this is PDK synced, I have reverted gitignore file changes.